tools/libraryowners/main.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"golang.org/x/mod/modfile"
+)
+
+const (
+	libraryOwnersPath = "build/ci/library_owners.json"
+	goModpath         = "go.mod"
+)
+
+func newGoMod() *modfile.File {
+	goModFile, err := os.ReadFile(goModpath)
+	if err != nil {
+		fmt.Printf("error reading %s: %v\n", goModpath, err)
+		os.Exit(1)
+	}
+
+	f, err := modfile.Parse(goModpath, goModFile, nil)
+	if err != nil {
+		fmt.Printf("error parsing %s: %v\n", goModpath, err)
+		os.Exit(1)
```

Comment:
fmt.Printf and then os.Exit(1) is no different from log.Fatal, by best practices I mean returning an erro and have the erro be handled 

IE: [Don't panic](https://github.com/golang/go/wiki/CodeReviewComments#dont-panic) 
Where log.Fatal or this approach is no sodifferent from a panic